index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@200;400;500;700&display=swap" rel="stylesheet">
+</head>
+<body class="main__container">
+    <!-- Creating the navigation menu bar -->
+  <header class="container_menu">
+    <nav class="container--nav">
+        <ul class="navegation--list">
+            <li class="navigation--option"><a>Portfolio</a></li>
+            <li class="navigation--option"><a>About</a></li>
+            <li class="navigation--option"><a>Contact</a></li>
+        </ul>
+        <div class="menu--image">
+            <img src="images\menuUnion.png" alt="Menu button"/>
+        </div>
+    </nav>
+  </header>
+  <!-- This section is the main part, containing heading and a brief description of my work-->
+  <main class="main__containerinfo">
+   <h1 id="titlePortfolio">Hey there. I’m Reynaldo. </h1>
+   <h2 id="developerPortfolio">I’m a software developer</h2>
+   <p id="descriptionPortfolio"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+    don’t hesitate to contact me.</p>
+  </main>
+
+  <section class="container__icons">
+    <div class="brand--icon">
```

Comment:
- [ ] please ensure that the color for your social media icons are the same as the provided design on figma :+1: 
- figma design
![sm-i](https://github.com/ReyAntonio12/Reynaldo--Portfolio/assets/89055502/e41fe20b-24ea-47ef-94ff-546cf2fd05b5)
- your design
![sm-ic](https://github.com/ReyAntonio12/Reynaldo--Portfolio/assets/89055502/46fb03a0-d333-4922-a4ba-b9b34deb3fb8)